src/_gettsim/transfers/rente.py

```diff
@@ -625,13 +674,28 @@ def _ges_rente_arbeitsl_altersgrenze(
         :ref:`ges_rente_params <ges_rente_params>`.
     ges_rente_regelaltersgrenze
         See :func:`ges_rente_regelaltersgrenze`.
+    _ges_rente_arbeitsl_altersgrenze_vertrauenss
+        See :func:`_ges_rente_arbeitsl_altersgrenze_vertrauenss`.
+    vertrauenss_arbeitsl
+        See basic input variable :ref:`vertrauenss_arbeitsl <vertrauenss_arbeitsl>`.
 
     Returns
     -------
     lowest full retirement age for unemployed.
 
     """
-    if birthdate_decimal < ges_rente_params["abolishment_cohort_rente_für_arbeitsl"]:
+    if (
+        int(birthdate_decimal)
+        <= ges_rente_params["cohort_range_arbeitsl_vertrauenss"]["last"]
```

Comment:
this should become 

```py
if birthyear in ges_rente_params["altersgrenze_arbeitsl_abschlagsfrei_vertrauensschutz"]:
```
or so.

The if-statements below can be replace by indexing into months (use `birthmonth`, too)